tools/topology/topology2/include/components/src_format_s32_convert_from_48k.conf

```diff
@@ -1,95 +1,34 @@
 #src format array
 			num_output_audio_formats 14
 
-			Object.Base.output_audio_format [
-				# 8khz output
-				{
-					out_rate                 8000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 11.025 khz output
-				{
-					out_rate                 11025
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
-				}
-				# 12khz output
-				{
-					out_rate                 12000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 16khz output
-				{
-					out_rate                 16000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 22.05khz output
-				{
-					out_rate                 22050
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 24khz output
-				{
-					out_rate                 24000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 32khz output
-				{
-					out_rate                 32000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 44.1khz output
-				{
-					out_rate                 44100
-					out_bit_depth            32
-					out_valid_bit_depth      32
-					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
-				}
-				# 48khz output
-				{
-					out_rate                 48000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 64khz output
-				{
-					out_rate                 64000
-					out_bit_depth            32
-					out_valid_bit_depth      32
-				}
-				# 88.2khz output
-				{
-					out_rate                 88200
-					out_bit_depth            32
-					out_valid_bit_depth      32
+			CombineArrays.Object.Base.output_audio_format [
+				{
+					out_rate [
+						8000
+						11025
```

Comment:
shouldn't we put this 11.025kHz to the next group?

Reply:
wonder why 22050 does not have obs setting @singalsu